package.json

```diff
@@ -179,6 +182,7 @@
     "package": "npm run bundle && jpm xpi && mv activity-streams.xpi dist/activity-streams-$npm_package_version.xpi",
     "travis": "npm run test",
     "prepush": "npm run test:lint && npm run yamscripts",
+    "storybook": "start-storybook -p 9001 -s ./data/content",
```

Comment:
Ah, so this is already in here! Should work without the require css line then, no?